city_scrapers/spiders/det_employment_solutions.py

```diff
@@ -0,0 +1,153 @@
+import logging
+from datetime import datetime
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE, BOARD, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class DetEmploymentSolutionsSpider(CityScrapersSpider):
+    name = "det_employment_solutions"
+    agency = "Detroit Employment Solutions Corporation"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.descmiworks.com/about-us/public-meetings/"]
+    curr_type = None
+
+    MEETING_TYPES = ["MWDB", "CEAC"]
+    MEETING_TYPE_TO_CSS_SELECTOR_MAP = {
+        "MWDB": "div.mwdb-meeting_wrapper_inner-wrapper",
+        # TODO: find selector for CEAC meetings
+        "CEAC": "div",
+    }
+
+    def set_current_meeting_type(self, item):
+        """Setter function for curr_meeting_type property."""
+        if not item:
+            self.curr_type = None
+        info = item.get()
+        # Right now there are only 2 parsable meeting types on the webpage
+        if "mwdb" in info:
```

Comment:
It might be a little easier to read if instead of setting `self.curr_type` here we passed the meeting type as a parameter to the other functions

Reply:
Good call! I like your suggestion much more than what I did!